### ISSUE TITLE
Encoding image batches

### ISSUE THREAD

**Issue by davidoort** [facebookresearch/r3m#25]
Hey, thanks a lot for open-sourcing this! I've seen that you guys have an example for a single image [here](https://github.com/facebookresearch/r3m/blob/main/r3m/example.py) but I'm wondering what kind of image dimensions can be passed to the pre-trained models and if they can work with image batches (e.g. (num_envs, 3, H, W)). Thanks!

**Comment by davidoort** [comment]
A side question, but also related to the configuration or specification of inputs and outputs is: can we control the output dimension of any of the pre-trained models?

**Comment by suraj-nair-1** [comment]
You can pass in any batch size. As for the image height/width we only tested it with the 224x224 size we use. I believe the network can take in other sizes, but it may have an impact on the performance. Anecdotally, some people have found that if using other image sizes its important to fine-tune. 

The output dim is fixed at 512 for the resnet18 and resnet34 and 2048 for the resnet50. If you want to resize it you can add another layer.
